src/server/database/update.js

```diff
@@ -53,11 +54,24 @@ function saveGraph(pcID, graphID, releaseID, existingGraph, newGraph,connection)
 function updateGraph(pcID, releaseID, cyJson, connection, callback) {
   let graphID = uuid();
 
+  let nodeSort = (n1, n2) => {
```

Comment:
So node ids are numbers?

You can also use lodash to sort by key I believe.

Reply:
Node id's are uuid strings

Reply:
`_.sortBy( array, val => val.comparableValue )`